docs/installation.rst

```diff
@@ -1,7 +1,41 @@
 Installation
 ============
 
-Requirements before install
+Using Nix
+---------
+
+By using the Nix package manager, you can install scikits-odes in one
+line. Of course you need to install `nix <https://nixos.org/nix/>`_
+first.
+
+    curl https://nixos.org/nix/install | sh
+
+And now you can start a python shell with scikits-odes (and numpy) ready for use.
+
+    nix-shell -I nixpkgs=~/nixpkgs \
+    -p python37Packages.scikits-odes \
+    -p python37Packages.numpy \
+    --run "python3"
+
+To prove that lapack is available (although the nix install will have
```

Comment:
I moved the line about running tests to the paragraph above. Here, I'd just write:

> For a quick check, run the following code:

Reply:
I believe I have incorporated your suggestions.